### ISSUE TITLE
v16.4 Fails to transcode webm

### ISSUE THREAD

**Issue by MrSmite** [paulpacifico/shutter-encoder#96]
OS: Windows 7
App version: 16.4

I used yt-dlp to download a youtube clip. It was downloaded as a segmented DASH file and then yt-dlp merged the audio / video tracks using ffmpeg to create a webm file. When loaded in shutter-encoder, I can play the clip by double clicking it and shutter-encoder will launch ffplay, but when I try to do any conversion except image, it fails without any errors in the log.

1. Load the downloaded webm
2. Select H264 -> mp4
3. Click "start function"

The progress goes to 100% almost instantly. I am left with a 0-byte file but ffmpeg is still running in process explorer at about 48% cpu. If I refresh windows explorer, the 0-byte file's size will increase but eventually ffmpeg will go to 0% cpu within about 30 seconds.

The console only shows the command parameter passed to ffmpeg, and the metadata in the webm file. There is no error code or description on any of the console tabs.

If I turn on the "display" option to watch the encoding, it runs for about 30 seconds and stops. The resulting file is unplayable and incomplete (it's 10 minutes but the result is only 2,349 KB). If I choose "image" then "png", it will properly convert the webm to a png sequence.

**Comment by MrSmite** [comment]
Some more information:

The problem appears to be files that have metadata consisting of :

```
HTTP://YOUTUBE.COM/STREAMING/O : Segment-Count: 162 / Segment-Durations-Ms: 539,538,539(r=1)
```

where the timecodes repeat several times. If you run this file through mkvtoolnix or something that can strip metadata, the file then works fine in shutter-encoder.

**Comment by paulpacifico** [comment]
This issue has been corrected with the v16.7
